docs/src/rules/no-restricted-imports.md

```diff
@@ -422,6 +518,14 @@ import isEmpty, { hasValue } from 'utils/collection-utils';
 
 :::
 
+To restrict the use of all Node.js core imports (via <https://github.com/nodejs/node/tree/master/lib>):
+
+```json
+    "no-restricted-imports": ["error",
+         "assert","buffer","child_process","cluster","crypto","dgram","dns","domain","events","freelist","fs","http","https","module","net","os","path","punycode","querystring","readline","repl","smalloc","stream","string_decoder","sys","timers","tls","tracing","tty","url","util","vm","zlib"
+    ],
+```
+
```

Comment:
I would leave this out. It's a very specific use case that it easy to figure out if you need it. (It would probably also be a better idea to use the `globals` package for this purpose.)

Reply:
i this means we can remove this section?

Reply:
Yes, I think we can remove this section. It's not necessary.

Reply:
Done!